lib/node_modules/@stdlib/assert/is-same-typed-array-like/test/test.js

```diff
@@ -0,0 +1,104 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Int8Array = require( '@stdlib/array/int8' );
+var Int16Array = require( '@stdlib/array/int16' );
+var isSameTypedArrayLike = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof isSameTypedArrayLike, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function returns `true` if provided two typed-array-like objects having the same contents', function test( t ) {
+	var x;
+	var y;
+
+	x = new Int8Array( [ 1.0, 2.0, 3.0 ] );
+	t.strictEqual( isSameTypedArrayLike( x, x ), true, 'returns expected value' );
+
+	x = new Int8Array( [ 1.0, 2.0, 3.0 ] );
+	y = new Int16Array( [ 1.0, 2.0, 3.0 ] );
+	t.strictEqual( isSameTypedArrayLike( x, y ), true, 'returns expected value' );
+
+	x = new Int8Array( [ 1.0, 2.0, 3.0 ] );
```

Comment:
Test statement specified for condition when function return `true`. You should not include the test which return `false`.

Reply:
This is not done yet. You need to create examples that follows test statement.

Reply:
ah i created the new tests but guess forgot to remove that from there thanks for pointing it out

Reply:
done!